speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/MediatorTests.cpp

```diff
@@ -251,3 +251,26 @@ void TestBoolArrMediator(bool p_env, bool p_car, bool p_human, bool p_interventi
 BEGIN_TEST_COMBINATORIAL(MediatorTests, DataCollectionSettings)
 bool booleans[] = {false, true};
 END_TEST_COMBINATORIAL5(TestBoolArrMediator, booleans, 2, booleans, 2, booleans, 2, booleans, 2, booleans, 2)
+
+/// @brief Tests if the TimeOut function returns the correct time out
+TEST(MediatorTests, TimeOutTest)
+{
+    Random random;
+    char path[256];
+    for (int j = 0; j < TEST_AMOUNT; j++)
+    {
+        SDAConfigMediator::ClearInstance();
+        ASSERT_TRUE(SetupSingletonsFolder());
+
+        int maxTimeMinutes = random.NextInt();
```

Comment:
You should use random.NextInt(minimum, maximum) so you do not get gigantic numbers or negative numbers. (Tho a test testing on those would also be good, testing edge cases is good)